x/gov/types/v1/params.go

```diff
@@ -419,5 +423,16 @@ func (p Params) ValidateBasic() error {
 		return fmt.Errorf("minimum initial deposit decrease ratio too large: %s", minInitialDepositDecreaseRatio)
 	}
 
+	burnDepositNoThreshold, err := sdk.NewDecFromStr(p.BurnDepositNoThreshold)
+	if err != nil {
+		return fmt.Errorf("invalid burnDepositNoThreshold string: %w", err)
+	}
+	if !burnDepositNoThreshold.IsPositive() {
```

Comment:
I would replace this check with the `burnDepositNoThreshold > 1 - threshold` && `burnDepositNoThreshold > 1 - lawThreshold` && && `burnDepositNoThreshold > 1 - amendmentThreshold`